Configure

```diff
@@ -240,6 +240,7 @@ my @disablables = (
     "autoalginit",
     "autoerrinit",
     "bf",
+    "bp",
```

Comment:
bp is pretty terse. OpenSSL historically used to operate on a limited supply of characters but this is no longer the case. How about "pairing"? (Same for the header, etc.)